README.md

```diff
@@ -7,16 +7,57 @@ Docker: Believe the Hype!
 Learn how to use docker.io containers to consistently deploy
 your apps on any infrastructure.
 
-
-## What? (*Quick* Introduction)
+## Why?
+Have you ever struggled to get your developer environment set up as a new user on
+an existing project? Or have you ever had difficulty reproducing a bug or error
+that another team member is experiencing? Or that only exists in your production
+environment but not in your development environment. If you have, then you have
+encountered some of the difficulties of not working in the same environment as
+your teammates on a project. Docker is the solution to these problems as well as
```

Comment:
Nice one mentioning bugs in production as well 👍 

I think that it may have changed the flow of the next sentence in the paragraph now though. 
Maybe you could update the next sentence to be:
"If you have, then you have encountered some of the difficulties that can arise when code is run in different environments."?

Reply:
Silly me! I missed that. Thanks @RobStallion - definitely need to change that 👍